.dotnet/tests/Samples/Images/Sample02_SimpleImageEdit.cs

```diff
@@ -14,23 +14,23 @@ public void Sample02_SimpleImageEdit()
             ImageClient client = new("dall-e-2", Environment.GetEnvironmentVariable("OpenAIClient_KEY"));
 
             string imagePath = Path.Combine("Assets", "edit_sample_image.png");
-            BinaryData imageBytes = BinaryData.FromBytes(File.ReadAllBytes(imagePath));
+            using FileStream inputImage = File.OpenRead(imagePath);
 
             string prompt = "An inflatable flamingo float in a pool";
 
             string maskPath = Path.Combine("Assets", "edit_sample_mask.png");
-            BinaryData maskBytes = BinaryData.FromBytes(File.ReadAllBytes(maskPath));
+            using Stream mask = File.OpenRead(maskPath);
```

Comment:
nit: For consistency, should both (image and mask) be FileStreams? Alternatively, both can be Streams.

Reply:
Ah, good call out regarding consistency in the sample.  I think it would be better to keep them both Streams, since we can't add the FileStream conveniences when the mask lives in options properties.  It's an interesting point, and depending on how consistent we want to be with conveniences, maybe there's an argument to be made for a second convenience overload that lets you pass the mask as a FileStream parameter to the method?  But the quickest thing to do right now is just to update the sample to use the Stream file/string fileName version in both places.  Thanks!

Reply:
Addressed.